### PR TITLE
DUOS-2168[risk=no] Delete DarTable.js

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [DUOS-2168](https://broadworkbench.atlassian.net/browse/DUOS-2168)

Simple PR, it just removed `DarTable.js` from the repo since it's long been replaced by `DarCollectionTable.js`

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/dar_table/DarTable.js [hunk 1]
```diff
@@ -1,258 +0,0 @@
...
-      setCurrentPage(currentPage);
-    }
-  };
-
-  //NOTE: method used in the confirmation modal
-  const createElection = async (darId, index) => {
-    if (!isNil(darId)) {
-      try{
-        //updatedElection is the new access election
-        const updatedElection = await Election.createDARElection(darId);
```
Comment (rushtong) [inline src/components/dar_table/DarTable.js:204]: This method is no longer used, we should remove it.

File: src/components/dar_table/DarTable.js [hunk 2]
```diff
@@ -1,258 +0,0 @@
...
-    //Use timeout to ensure that the rebuild fires after the child component has rendered
-    //clear timeout after callback function fires to avoid side-effects
-    ReactTooltip.rebuild();
-  }, [currentPage, tableSize, filteredList]);
-
-  const openModal = useCallback(async(darInfo) => {
-    let darData = darInfo.data;
-    if (!isNil(darData)) {
-      setShowModal(true);
-      darData.researchType = DataUseTranslation.generateResearchTypes(darData);
```
Comment (rushtong) [inline src/components/dar_table/DarTable.js:160]: This method is no longer used, we should remove it.

File: src/components/dar_table/DarTable.js [hunk 3]
```diff
@@ -1,258 +0,0 @@
...
-import { useState, useEffect, useCallback } from 'react';
-import { div, h, span } from 'react-hyperscript-helpers';
-import { Election, User } from '../../libs/ajax';
-import { DataUseTranslation } from '../../libs/dataUseTranslation';
-import { Notifications, getDatasetNames } from '../../libs/utils';
-import { Styles } from '../../libs/theme';
-import DarModal from '../modals/DarModal';
-import PaginationBar from '../PaginationBar';
-import ConfirmationModal from '../modals/ConfirmationModal';
-import DarElectionRecords from './DarElectionRecords';
```
Comment (rushtong) [inline src/components/dar_table/DarTable.js:11]: This looks like it was the only usage of this class, we should remove it.

Comment (rushtong) [reply src/components/dar_table/DarTable.js:11]: It looks like `DarTableActions` is only used by `DarElectionRecords`, so we could get rid of both of these.

**Review by rushtong — COMMENTED** [review]
This is looking like a good start, but I think there's more cleanup that can be done - see comments inline.

**Review by rushtong — APPROVED** [review]
Wow, great work! 👍🏽

**Review by connorlbark — APPROVED** [review]
looks good to me!

**Pull request merged**